the-blue-alliance-ios/Services/RemoteConfigService.swift

```diff
@@ -0,0 +1,48 @@
+import Crashlytics
+import FirebaseRemoteConfig
+import Foundation
+
+class RemoteConfigService {
+
+    private var remoteConfig: RemoteConfig
+    internal var retryService: RetryService
+
+    init(remoteConfig: RemoteConfig, retryService: RetryService) {
+        self.remoteConfig = remoteConfig
+        #if DEBUG
+        if let configSettings = RemoteConfigSettings(developerModeEnabled: true) {
+            self.remoteConfig.configSettings = configSettings
+        }
+        #endif
+        self.retryService = retryService
+
+        remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults")
+    }
+
+    internal func fetchRemoteConfig(completion: ((_ error: Error?) -> Void)? = nil) {
```

Comment:
Make sure we test this zz

Reply:
Probably not going to test this - RemoteConfig isn't really mockable because it's `init` is marked as unavailable